docs/reference/search/search-fields.asciidoc

```diff
@@ -0,0 +1,30 @@
+[discrete]
+[[search-fields]]
+=== Return fields in a search
+
+By default, each hit in the search response includes the document
+<<mapping-source-field,`_source`>>, which is the entire JSON object that was
+provided when indexing the document. Often only certain fields are required in
+the response. In this case you can use
+<<request-body-search-source-filtering,source filtering>> to restrict what
+parts of the source are returned.
```

Comment:
Reword for easier reading. If you keep this as-is, I would add a comma to `In this case,`.
```suggestion
If you only need certain fields in the search API's response, you can use
<<request-body-search-source-filtering,source filtering>> to restrict what
parts of the source are returned.
```